include/node.tcc

```diff
@@ -397,19 +397,25 @@ bool mpm::Node<Tdim, Tdof, Tnphases>::apply_absorbing_constraint(
           Eigen::Matrix<double, Tdim, 1> absorbing_force_;
           double force0 = 0.0;
           double force1 = 0.0;
-          if (position == mpm::Position::Corner) {
-            force0 = 0.5 * h_min * absorbing_traction_(0);
-            force1 = 0.5 * h_min * absorbing_traction_(1);
-          } else if (position == mpm::Position::Bottom) {
-            force0 = 0.5 * h_min * absorbing_traction_(0);
-            force1 = 1.0 * h_min * absorbing_traction_(1);
-          } else if (position == mpm::Position::Side) {
-            force0 = 1.0 * h_min * absorbing_traction_(0);
-            force1 = 0.5 * h_min * absorbing_traction_(1);
-          } else {
-            throw std::runtime_error("Invalid absorbing boundary position");
+          switch (position) {
+            case mpm::Position::Corner:
+              force0 = 0.5 * h_min * absorbing_traction_(0);
+              force1 = 0.5 * h_min * absorbing_traction_(1);
+              break;
+            case mpm::Position::Bottom:
+              force0 = 0.5 * h_min * absorbing_traction_(0);
+              force1 = 1.0 * h_min * absorbing_traction_(1);
+              break;
+            case mpm::Position::Side:
+              force0 = 1.0 * h_min * absorbing_traction_(0);
+              force1 = 0.5 * h_min * absorbing_traction_(1);
+              break;
```

Comment:
@kks32 @cgeudeker I am longer convinced this scheme is correct. Now, my understanding is that the "side" and "bottom" nodes should simply be
```
force = h_min * absorbing_traction_
```
I was incorrectly thinking about the applied tractions in terms of a frame-like problem instead of a continuum problem. 

If others agree - I will update this section (& greatly simplify whats going on here).

Reply:
Fixed